.github/workflows/ruff.yml

```
@@ -7,4 +7,4 @@ jobs:
       - uses: actions/checkout@v4
       - uses: astral-sh/ruff-action@v2
         with:
-          args: "format --check"
\ No newline at end of file
+          args: "check"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)